sdk/identity/Azure.Identity/TROUBLESHOOTING.md

```diff
@@ -1,270 +1,222 @@
-installed, please follow the instructions [here](https://docs.microsoft.com/powershell/azure/install-az-ps)
-to install it for your platform and then try running the credential again.
+`CredentialUnavailableException`
+| Error Message |Description| Mitigation |
+|---|---|---|
+|PowerShell isn’t installed.|No local installation of PowerShell was found.|Ensure that PowerShell is properly installed on the machine. Instructions for installing PowerShell can be found [here](tps://docs.microsoft.com/powershell/scripting/install/installing-powershell).|
+|Az.Account module >= 2.2.0 isn’t installed.|The Az.Account module needed for authentication in Azure PowerShell isn’t installed.|Install the latest Az.Account module. Installation instructions can be found [here](https://docs.microsoft.com/powershell/azure/install-az-ps).|
+|Please run 'Connect-AzAccount' to set up account.|No account is currently logged into Azure PowerShell.|<ul><li>Login to Azure PowerShell using the Connect-AzAccount command. More instructions for authenticating Azure PowerShell can be found [here](https://docs.microsoft.com/powershell/azure/authenticate-azureps)</li><li>Validate that Azure PowerShell can obtain tokens. See [below](#verifying-azure-powershell-can-obtain-tokens) for instructions.</li></ul>|
 
-#### Azure account not logged in.
+#### __Verifying Azure PowerShell can obtain tokens__
+You can manually verify that Azure PowerShell is properly authenticated, and can obtain tokens. First use the `Get-AzContext` command to verify the account which is currently logged in to the Azure CLI. 
 
-The `AzurePowerShellCredential` utilizes the current logged in Azure user in Azure Powershell to fetch an access token. You need to login to your account in Azure
-Powershell via `Connect-AzAccount` command. You can further read instructions
-to [Sign in with Azure Powershell](https://docs.microsoft.com/powershell/azure/authenticate-azureps). Once logged in try running the credential again.
+```
+PS C:\> Get-AzContext
 
-#### Deserialization error.
+Name                                     Account             SubscriptionName    Environment         TenantId
+----                                     -------             ----------------    -----------         --------
+Subscription1 (xxxxxxxx-xxxx-xxxx-xxx... test@outlook.com    Subscription1       AzureCloud          xxxxxxxx-x...
+```
 
-The `AzurePowerShellCredential` was able to retrieve a response from the Azure Powershell when attempting to get an access token but failed to parse that response.
-In your local powershell window, run the following command to ensure that Azure PowerShell is returning an access token in correct format.
+Once you've verified Azure PowerShell is using correct account, you can validate that it’s able to obtain tokens for this account.
 
-```pwsh
-Get-AzAccessToken -ResourceUrl "<Scopes-Url>"
+```bash
+Get-AzAccessToken -ResourceUrl "https://management.core.windows.net"
```

Comment:
should we specify "<Scopes-Url>"  instead of being specific about "https://management.core.windows.net"

Reply:
I think Scopes-Url is a misnomer here. You need the service's resource string not the scope, which would be "https://management.core.windows.net/.default". Regardless, the reason I chose to use a specific resource here is because this is intended to test if powershell is in fact authenticated and can get tokens. I don't want to make the user have to figure out what scopes their app is are requesting, and covert those to resource strings, just to validate this.